django/middleware/security.py

```diff
@@ -43,4 +45,20 @@ def process_response(self, request, response):
         if self.xss_filter:
             response.setdefault('X-XSS-Protection', '1; mode=block')
 
+        if self.referrer_policy:
+            # Support a comma-separated string or iterable of values to allow fallback.
+            response.setdefault('Referrer-Policy', ','.join(
+                [v.strip() for v in self.referrer_policy.split(',')]
```

Comment:
Do we need to do this? it is confusing (at least for me) to split and then join by `','`, maybe:
```python
response.setdefault(
    'Referrer-Policy',
    self.referrer_policy if isinstance(self.referrer_policy, str) else ','.join(self.referrer_policy),
)
```

Reply:
I don't think that this header is whitespace-sensitive. :thinking:

Reply:
I think I'd rather leave this. Leading/trailing whitespace is allowed but in-between multiple values... Grrr. 

https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2